binddump.c

```diff
@@ -267,13 +267,13 @@ void print_info(dyld_t *dylds, uintptr_t ndyld) {
         printf("%s\n", dylds[i].name);
         if(dylds[i].nnon_syms) {
             printf("    Non-Lazy:\n");
-            for(int j = 0; j<dylds[i].nnon_syms; j++) {
+            for(int j = 0; (unsigned)j<dylds[i].nnon_syms; j++) {
                 printf("      %s\n", dylds[i].non_symbols[j]);
             }
         }
         if(dylds[i].nlazy_syms) {
             printf("    Lazy:\n");
-            for(int j = 0; j<dylds[i].nlazy_syms; j++) {
+            for(int j = 0; (unsigned)j<dylds[i].nlazy_syms; j++) {
```

Comment:
This comment is for both this change and the change on line 270.

Casting (in general) should only be done when there is no way around it.  Especially casting between `signed` (i.e. int) and `unsigned` because it can cause some unexpected behavior.  The best thing to do here would be to change the initial type of `j` to an `unsigned` type instead of casting it later on as there is no need for `j` to ever be `signed` in this case.